wolvic/browser/vr/wvr_device.cc

```diff
@@ -119,7 +119,9 @@ void WvrDevice::OnWvrGlInitializationComplete(
       &WvrManager::StartWebXRPresentation,
       wvr_thread_->GetWvrManager()->GetWeakPtr(), std::move(options),
       CreateMainThreadCallback(
-          base::BindOnce(&WvrDevice::OnStartPresentResult, GetWeakPtr()))));
+          base::BindOnce(&WvrDevice::OnStartPresentResult, GetWeakPtr())),
+      CreateMainThreadCallback(base::BindOnce(
+          &WvrDevice::StopPresenting, GetWeakPtr(), base::NullCallback()))));
```

Comment:
We should be more consistent with naming, `OnStartPresentingResult` vs `StopPresenting`.

Reply:
Renamed OnStartPresentingResult to OnStartPresenting to be consistent